catalog/app/website/pages/AwsMarketplace/Features.tsx

```diff
@@ -0,0 +1,89 @@
+import cx from 'classnames'
+import * as React from 'react'
+import * as M from '@material-ui/core'
+
+import dataCommons from './dataCommons.svg'
+import dataVersionControl from './dataVersionControl.svg'
+import visualizations from './visualizations.svg'
+
+const useFeatureStyles = M.makeStyles((t) => ({
+  root: {
+    color: t.palette.text.primary,
+    background: `50% 0 / 78px 78px no-repeat`,
+    padding: '100px 22px',
+  },
+  heading: {
+    textAlign: 'center',
+    fontSize: '23px',
+    lineHeight: '32px',
+    marginBottom: '10px',
+  },
+  content: {
+    fontSize: '13px',
+    lineHeight: '32px',
+  },
+}))
+
+interface FeatureProps {
+  children: React.ReactNode
+  className?: string
+  heading: string
+  logo: string
+}
+
+function Feature({ children, className, heading, logo }: FeatureProps) {
+  const classes = useFeatureStyles()
+  const imgStyles = React.useMemo(() => ({ backgroundImage: `url("${logo}")` }), [logo])
+  return (
+    <div className={cx(classes.root, className)} style={imgStyles}>
+      <M.Typography className={classes.heading}>{heading}</M.Typography>
+      <M.Typography className={classes.content}>{children}</M.Typography>
+    </div>
+  )
+}
+
+const useStyles = M.makeStyles({
+  root: {
+    columnGap: '46px',
+    display: 'grid',
+    gridTemplateColumns: '1fr 1fr 1fr',
+    padding: '55px 0',
+  },
+})
+
+interface FeaturesProps {
+  className?: string
+}
+
+export default function Features({ className }: FeaturesProps) {
+  const classes = useStyles()
+  return (
+    <div className={cx(classes.root, className)}>
+      <Feature heading="Dataset Version Control" logo={dataVersionControl}>
+        Quilt Data provides dataset immutability by allowing users to create and share
+        datasets that are sealed and cryptographically verifiable, ensuring that the data
+        cannot be altered after it has been shared. This helps to maintain the integrity
+        and reliability of the data and ensures that users can trust the data
+        they are working with.
+      </Feature>
+      <Feature heading="Data Commons" logo={dataCommons}>
```

Comment:
```suggestion
      <Feature heading="Social Knowledge" logo={dataCommons}>
```
Data Commons is pretty loaded. You cold try "Data Mesh", idk.